cli/lib/context/context_gh_create_pr.dart

```diff
@@ -6,14 +6,16 @@ extension ContextGhCreatePr on Context {
     String baseBranch,
     String headBranch,
   ) {
+    final replacedBaseBranch = getReplacedBaseBranch(baseBranch);
```

Comment:
Let's move this logic right after this line instead: https://github.com/TarasMazepa/stax/blob/d5062340f2f66920673c1c6bac279755d46e44f4/cli/lib/command/internal_command_commit.dart#L97